entrypoint.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+## fix for newer debian versions
+[ -d /dev/net ] || mkdir -p /dev/net
+[ -c /dev/net/tun ] || mknod /dev/net/tun c 10 200
```

Comment:
Just curious, if `modprobe tun` does create this dev file.

Reply:
https://superuser.com/a/598412

modprobe will always return module not found :

$ sudo modprobe tun
FATAL: Module tun not found.
Finally, I found that tun is not a module but built in kernel, so what I do to solve was created the missing dir and nod:

$ sudo mkdir /dev/net
$ sudo mknod /dev/net/tun c 10 200
And then openvpn can find and use the tun device.